garage/tf/policies/base2.py

```diff
@@ -88,6 +95,14 @@ def terminate(self):
         """Clean up operation."""
         pass
```

Comment:
What's this function for? Why pass?

Reply:
I copy from the existing policy base class from `garage.tf.policies.base`. If it is not used anywhere, we can remove that.